rolling/ci-nightly-zenoh.yaml

```diff
@@ -0,0 +1,59 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  ROS_PYTHON_VERSION: '3'
+build_tool: colcon
+build_tool_args: '--cmake-args --no-warn-unused-cli'
+build_tool_test_args: '--retest-until-pass 2 --ctest-args -LE xfail --pytest-args -m "not xfail"'
+jenkins_job_label: ci-agent
+jenkins_job_priority: 50
+jenkins_job_schedule: 15 23 * * *
+jenkins_job_timeout: 300
+jenkins_job_weight: 4
+package_selection_args: '--packages-ignore foonathan_memory_vendor --packages-ignore-regex .*connext.* .*cyclonedds.* .*fastrtps.* .*fastdds.*'
```

Comment:
This needs to be updated to build `rosidl_typesupport_fastrtps_c{,pp}`. Evidently Zenoh needs that.